app/views/docs/authentication-security.phtml

```diff
@@ -0,0 +1,96 @@
+            </td>
+            <td>
+                Apple
+            </td>
+            <td>
+                Uses a session cookie stored in <b>UserDefaults</b>.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <img src="/images-ee/platforms/android.svg" alt="Logo" height="30" class="force-light sdk-logo margin-start margin-end" />
+                <img src="/images-ee/platforms/dark/android.svg" alt="Logo" height="30" class="force-dark sdk-logo margin-start margin-end" />
+            </td>
+            <td>
+                Android
+            </td>
+            <td>
+                Uses a session cookie stored in <b>SharedPreferences</b>.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<h2><a href="/docs/authentication-security#limits" id="limits">Session Limits</a></h2>
+<p>In Appwrite versions 1.2 and above, you can limit the number of active sessions created per user to prevent the accumulation of unused but active sessions. New sessions created by the same user past the session limit deletes the oldest session.</p>
+
+<p>You can change the session limit in the <b>Security</b> tab of the Auth Service in your Appwrite Console. The default session limit is 10 with a maximum configurable limit of 100.</p> 
+
+<h2><a href="/docs/authentication-security#security" id="security">Security</a></h2>
+
+<p>
+    Security is very important to protect users' data and privacy. Appwrite uses a <a href="/docs/permissions">permissions model</a> coupled with user sessions to ensure users only have access to certain information based on the permissions. With Appwrite services, including databases and storage, access is granted at the collection, bucket, document, or file level. This access is consistent across access to these items in relation to document access, file access, and real-time events.
+</p>
+
+<h2><a href="/docs/authentication-security#password-history" id="password-history">Password History</a></h2>
+<p>Password history prevents users from reusing recent passwords. This protects user accounts from security risks by enforcing a new password everytime it's changed.</p>
+
+<p>Password history can be enabled in the Auth service's <b>Settings</b> on the Appwrite console. You can choose how many previous passwords to remember and block users from reusing them.</p>
+
+<h2><a href="/docs/authentication-security#password-dictionary" id="password-dictionary">Password Dictionary</a></h2>
+<p>Password dictionary protects users from using bad passwords. It compares the user's password to the <a href="https://github.com/danielmiessler/SecLists/blob/master/Passwords/Common-Credentials/10k-most-common.txt">10,000</a> most common passwords and throws a warning if there's a match. Together with <a href="/docs/rate-limits">rate limits</a>, password dictionary will significantly reduce the chance of a malicious actor from guessing user passwords.</p>
```

Comment:
Warning or Error? Because we throw error?

Reply:
error